pyg_lib/testing.py

```diff
@@ -52,9 +54,12 @@ def get_sparse_matrix(
     name: str,
     dtype: torch.dtype = torch.long,
     device: Optional[torch.device] = None,
-) -> Tuple[Tensor, Tensor]:
+    return_csc: Optional[bool] = False,
+) -> Tuple[Tensor, Tensor, Optional[Tensor], Optional[Tensor]]:
```

Comment:
This return type is a little confusing to me. Perhaps we can provide a separate utility function that handles CSC conversion?

Reply:
According to the [Matthias comment](https://github.com/pyg-team/pyg-lib/pull/79#discussion_r960343916) it has been removed.